css/result.css

```diff
@@ -0,0 +1,19 @@
+@import url("./questions.css");
+
+.result-wrapper {
+  display: flex;
+  flex-direction: column;
+  gap: 6.4rem;
+}
+
+.quiz-options button.right-answer {
+  background-color: var(--correct-color);
+}
+
+.quiz-options button.wrong-answer {
+  background-color: var(--incorrect-color);
+}
+
+.quiz-options button.disabled {
+  pointer-events: none;
+}
```

Comment:
Didn't get why you have referred to the classes of buttons as button.disabled instead of mentioning it as disabled-button and similarly with the right and wrong answer buttons?Can you explain ?

Reply:
I added a disabled class as I was planning on controlling it via code later - will change it if I change my mind

Reply:
Thanks for clarifying @Snigdha17